client/allocdir/alloc_dir_windows.go

```diff
@@ -3,6 +3,15 @@ package allocdir
 import (
 	"errors"
 	"os"
+	"path/filepath"
+)
+
+var (
+	//Path inside container for mounted directory that is shared across tasks in a task group.
+	SharedAllocContainerPath = filepath.Join("c:\\", SharedAllocName)
```

Comment:
Why are we hardcoding `c:\\` here? I understand that this is the default, but users can specify any other drive such as `D`, `E`, etc as their Data Dir. Is this going to work in all cases?